### DISCUSSION TITLE
Can you download new content only from where fansly scraper left off?

### DISCUSSION THREAD

**Discussion by deadbeatdandylyon** [Avnsx/fansly#103] [Q&A]
Okay so issue is, is there any way to continue the downloading from where the "Fansly Scraper" left off. 

Meaning if I scraped a creator with the Fansly Scraper can I configure the new Fansly Downloader to pick up where I left off or will I have to redownload everything yet again?

**Comment by deadbeatdandylyon** [comment]
I do know that there is a function called utilise_duplicate_threshold = True however although it is set to true its still downloading duplicate files and not registering that the files already downloaded

**Comment by Avnsx** [comment]
What you're refering to as downloading duplicate files, is most likely the 720p versions & then the newest UHD versions of the same content. Just download all the files into fresh folders with the new version.

**Comment by deadbeatdandylyon** [comment]
so there is no way for the system to just pick up where the other downloader stopped, have to re download everything gotcha
